### ISSUE TITLE
Implement `TrackAddButton`

### ISSUE THREAD

**Issue by adamnovak** [vgteam/sequenceTubeMap#227]
Create a `TrackAddButton` control as defined in #205. This should be a wide button with a "+" icon and appropriate tooltips and accessibility properties. When clicked, it should call the function in its `onClick` prop.
